extensions/anki/notes.md

```diff
@@ -0,0 +1,4 @@
+### Features
+
+- Configure Anki
+  - Need an option to cofigure the path to the Anki collection
```

Comment:
Is this file still needed?

Reply:
Thanks for pointing this out. It is not; will remove now.

Reply:
Removed ✅ 

Thanks!